external-docs/docs/glossary/index.md

```diff
@@ -1,3 +1,70 @@
+
+### Dimension
+The dimension components serve to identify the observations. A set of values for all the dimension components is sufficient to identify a single observation. Examples of dimensions include the time to which the observation applies, or a geographic region which the observation covers<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
+
+### Literal
+Basic values such as strings, dates, booleans, and numbers that can only be used in the object position of an RDF triple. To allow correct parsing and interpretation, literals are best associated with a datatype<sup>[[8]](http://www.proxml.be/losd/semcubes.html)</sup>.
+
+ ### Measure
+The measure components represent the phenomenon being observed<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
+
+### Observation
+A data point which can be characterised by a dimension that defines what the data point applies to (e.g. time, area, gender) along with metadata describing what has been measured (e.g. economic activity, population), how it was measured and how the data point is expressed (e.g. units, multipliers, status)<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
+
+### Population Characteristic
+A concept to describe the set of objects that information is to be obtained about in a statistical survey. For example, the Population of adults in the Netherlands based on the Unit Type of persons<sup>[[5]](https://sdmx.org/wp-content/uploads/SDMX_Glossary_version_2_1-Final-2.docx)</sup>.
+
+### Resource
+Used in a general sense for whatever might be identified by a URI.  Familiar examples include an electronic document, an image, a source of information with a consistent purpose and a collection of other resources<sup>[[9]](https://datatracker.ietf.org/doc/html/rfc3986)</sup>.
+
+### Tidy Data
+A format for laying out data sets. A tidy data set is arranged such that each dimension is a single column and each observation a single row. For more information, see Hadley Wickham's paper on this topic<sup>[[6]](https://www.jstatsoft.org/index.php/jss/article/view/v059i10/v59i10.pdf)</sup>.
+
+### Unit
+A quantity or increment by which something is counted or described, such as kg, mm, °C, °F, monetary units such as Euro or US dollar, simple number counts or index numbers. The unit of measure in connection with the unit multiplier, provides the level of detail for the observation value of the dimension<sup>[[3]](https://sdmx.org/wp-content/uploads/SDMX_Glossary_version_2_1-Final-2.docx)</sup>.
+
+### URI
+Uniform Resource Identifier – URIs are used to name the entities and concepts so that consumers of the data can look-up those URIs to get more information, including links to other related URIs. Note that a URL (Uniform Resource Locator) is a type of URI<sup>[[2]](https://www.w3.org/TR/vocab-data-cube/#intro-rdf)</sup>.\
```

Comment:
`- URIs are identifiers which distinguish resources from one another.`

I'd skip the bit about being able to look up URIs to get more information.

Reply:
Altered definition